cartesi-rollups/index.md

```diff
@@ -23,9 +23,11 @@ Let's delve into the workings of a Cartesi Rollup at a high level.
 ![img](../static/img/v1.3/overview.jpg)
 
 
-At its core, the Cartesi Rollup executes the Cartesi Machine - a robust RISCV deterministic emulator running Linux OS - fueled by ordered inputs and custom application code. Inputs from the data-availability layer are read by the Cartesi Node, where the Cartesi Machine processes them and generates outputs. After the optimistic rollup dispute window passes, these outputs are verifiable and possibly executable on the settlement layer.
+At its core, the Cartesi Rollup executes the Cartesi Machine - a robust RISCV deterministic emulator running Linux OS - fueled by ordered inputs and custom application code. Inputs from the data-availability layer are read by the Cartesi Node, where the Cartesi Machine processes them and generates outputs. After the optimistic rollup dispute window passes, these outputs are verifiable and possibly executable on the settlement layer. 
 
-In its simplest configuration, the Cartesi Rollup framework seamlessly integrates with Ethereum, acting as the sole data availability, consensus, and settlement layer. Input transactions are directed to specific smart contracts, where the dApp code operates to produce outputs. Following the settlement period, these outputs are verified and may also be executed on Ethereum, as required.
+The Cartesi Rollup architecture is application-specific, meaning each dApp within Cartesi's ecosystem runs on its unique, high-performance rollup app chain with its CPU. This prevents resource sharing with other dApps, significantly enhances computational scalability, and preserves the strong security guarantees of the underlying base layer.
+
+The Cartesi Rollup framework seamlessly integrates with Ethereum, acting as the sole data availability, consensus, and settlement layer in its most straightforward configuration. Input transactions are directed to specific smart contracts, where the dApp code operates to produce outputs. Following the settlement period, these outputs are verified and may be executed on Ethereum.
```

Comment:
I think we should revisit this whole paragraph. Let point a few things I see here:
- Ethereum is not the sole option for deployment, but EVM
- "Input transactions are directed" it seems like an automated thing, but the transaction should be pointing to the right place to begin with.
- "*where* the dApp code operates to produce outputs"; Where is "where" meaning here? The solidity smart contracts?
- Overall, I think there is too little information being transmitted in this whole paragraph. 


Ok, we explained that App Specific means having its own 'chain'. Briefly we mention that follows with some nice advantages. Now we are missing adding a bit of the security guarantees.  Here, at a high level, I think we should explain that as an App Specific *optimistic* rollup, you only need 1 honest actor to ensure the correct result/settlement. Through the _verification game_, disputes guarantees fair results even when there are collusion of every other actor against the honest one.

Reply:
Check this out:

```
The Cartesi Rollup framework tailors each DApp with its unique app chain and CPU, enhancing security by linking the dedicated consensus of these rollups directly to the base layer. This setup allows validators —  permissioned or not, to uphold the security standards of the base layer, ensuring that any honest validator can enforce the correct result with its support. The framework aligns closely with the base layer, which serves as the primary platform for data availability, consensus, and settlement. dApp transactions are processed through specific smart contracts to generate verified outputs that, after a settlement period, can be executed on the base layer. 
```

Reply:
:100: good to go

Reply:
All changes pushed. I believe we are good to approve/merge now